cmd/sqlflowserver/main_test.go

```diff
@@ -1616,7 +1616,8 @@ func getUniqueID() int {
 	return uniqueID
 }
 
-func caseXGBoostFeatureColumnImpl(t *testing.T, table string, label string, selectColumns string, columnClauses string, nclasses int, nworkers int, isPai bool) {
+func caseXGBoostFeatureColumnImpl(t *testing.T, table string, label string, selectColumns string, columnClauses string, nclasses int, nworkers int, isPai bool,
+	skipExplain bool) {
```

Comment:
Because XGBoost explanation uses `pandas.DataFrame` . See [here](https://github.com/sql-machine-learning/sqlflow/pull/2416/files#diff-9c0c52328de10b86c937f3425060d52dR53). It does not support `INDICATOR` now. If we want to support `INDICATOR` in XGBoost `INDICATOR`, we should separate each element in the one hot vector to several columns of `pandas.DataFrame`. I would fix it in the next PR. Marked.